test/doc.go

```diff
@@ -0,0 +1,6 @@
+// Package test contains integration tests with fake Slack and Kubernetes APIs.
```

Comment:
This package cannot be deleted until the true E2E tests from this PR run on CI. 
I would like to run these newly created tests both on PR and `develop` branch. As the BotKube Tester has really low permissions, I don't see any security issue in such approach. Of course, we will have a separate test workspace configured only for running these tests.